### PR TITLE
Linkify 'see also' sections

### LINKED ISSUE #150562: Some doc comments refer to unknown, or out of scope, code identifiers.
Here's the main list. Some just need to use backticks instead of square brackets. Some should just not have square brackets. Some are typos. Some need broader textual refactoring:

- [x] * `DragGestureRecognizer.onCancel` - property referencing `allowedButtonsFilter`.
- [x] * `DragGestureRecognizer.onDown` - property referencing `allowedButtonsFilter`.
- [x] * `DragGestureRecognizer.onEnd` - property referencing `allowedButtonsFilter`.
- [x] * `DragGestureRecognizer.onStart` - property referencing `allowedButtonsFilter`.
- [x] * `DragGestureRecognizer.onUpdate` - property referencing `allowedButtonsFilter`.
- [x] * `DoubleTapGestureRecognizer.onDoubleTap` - property referencing `allowedButtonsFilter`.
- [x] * `DoubleTapGestureRecognizer.onDoubleTapCancel` - property referencing referencing `allowedButtonsFilter`.
- [x] * `DoubleTapGestureRecognizer.onDoubleTapDown` - property referencing `allowedButtonsFilter`.
- [x] * `TapGestureRecognizer` - class referencing `allowedButtonsFilter`.
- [x] * `Badge.backgroundColor` - property referencing `ColorScheme.errorColor`.
- [x] * `ButtonThemeData.padding` - property referencing `button`.
- [x] * `ButtonThemeData.shape` - property referencing `button`.
- [x] * `InkSparkle` - class referencing `containedInkWell`.
- [x] * `InkSparkle` - class referencing `rectCallback`.
- [x] * `InkSparkle.splashFactory` - static const field referencing `turbulenceSeed`.
- [x] * `InkSparkle.constantTurbulenceSeedSplashFactory` - static const field referencing `turbulenceSeed`.
- [x] * `Switch.adaptive` - constructor referencing `ThemeData.adaptations`.
- [x] * `TextButton.defaultStyleOf` - method referencing `padding`.
- [x] * `Adaptation.adapt` - method referencing `ThemeData.adaptations`.
- [x] * `UndoManager` - Class referencing `canRedo`.
- [x] * `UndoManager` - Class referencing `canUndo`.
- [x] * `ContextMenuController.markNeedsBuild` - method referencing `contextMenuBuilder`.
- [x] * `SliverFadeTransition` - class referencing `sliver`.

And some just refer to not-properly-prefixed elements. These can be updated with a prefix:

- [x] * `ImageInfo.clone` - method referencing `Image.clone`.
- [x] * `ImageInfo.isCloneOf` - method referencing `Image.isCloneOf`.
- [x] * `RawImage` - class referencing `Image.dispose`.
- [x] * `RawImage.image` - property referencing `Image.dispose`.

And some just refer to not-imported elements. These can be updated with `@docImport`:

- [x] * `DiagnosticsNode.toTimelineArguments` - method referencing `Timeline.startSync`.
- [x] * `compute` - top-level function referencing `Isolate.run`.

### PR DESCRIPTION
**Pull request by goderbauer**
Follow-up to https://github.com/flutter/flutter/pull/150540.

Fixes https://github.com/flutter/flutter/issues/150562.

Part of https://github.com/flutter/flutter/issues/150800.

### CODE REVIEW

**Comment by goderbauer** [comment]
@srawlins Is there any way to quickly check that all these references continue to work with the work you're doing for `@docImport`?

**Comment by srawlins** [comment]
Hmm, no, there isn't really. :/ I'm just crossing my fingers, and looking to CI to see whether I _introduced_ any regressions. :/ Not a good verification state / process yet.

File: packages/flutter/lib/src/gestures/recognizer.dart [hunk 1]
```diff
@@ -178,7 +179,7 @@ abstract class GestureRecognizer extends GestureArenaMember with DiagnosticableT
   ///
   /// Defaults to all buttons.
   /// {@endtemplate}
-  final AllowedButtonsFilter _allowedButtonsFilter;
+  final AllowedButtonsFilter allowedButtonsFilter;
```
Comment (srawlins) [inline packages/flutter/lib/src/gestures/recognizer.dart:182]: Nice 😁

File: packages/flutter/lib/src/material/switch.dart [hunk 2]
```diff
@@ -137,10 +137,10 @@ class Switch extends StatelessWidget {
   /// is iOS or macOS, otherwise a Material Design switch is created.
   ///
   /// To provide a custom switch theme that's only used by this factory
-  /// constructor, add a custom `Adaptation<SwitchThemeData>` class to
-  /// `ThemeData.adaptations`. This can be useful in situations where you don't
-  /// want the overall [ThemeData.switchTheme] to apply when this adaptive
-  /// constructor is used.
+  /// constructor, pass a custom `Adaptation<SwitchThemeData>` class to the
+  /// `adaptations` parameter of [ThemeData]. This can be useful in situations
```
Comment (srawlins) [inline packages/flutter/lib/src/material/switch.dart:141]: 👍

**Review by srawlins — APPROVED** [review]
🙏 Thanks for looking into these!

**Pull request merged**